api/src/main/java/org/hawkular/alerts/api/services/AlertsCriteria.java

```diff
@@ -293,31 +295,27 @@ public void setThin(boolean thin) {
     }
 
     public boolean hasAlertIdCriteria() {
-        return null != alertId
-                || (null != alertIds && !alertIds.isEmpty());
+        return null != alertId || !isEmpty(alertIds);
     }
 
     public boolean hasSeverityCriteria() {
-        return null != severity
-                || (null != severities && !severities.isEmpty());
+        return null != severity || !isEmpty(severities);
```

Comment:
-1
severity is a Severity object, so, I don't see value wrapping a severity != null inside a isEmpty() on this case.